README.md

```diff
@@ -1 +1,157 @@
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 Hello-Microverse <a name="about-project"></a>
+
+Hello-Microverse is an introductory project to practice my Github flow skills and getting familiar working with linters.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+It contains:
+
+- A HTML file with the text "Hello Microverse!"
+- A CSS file 
+- A gitignore and README files.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](http://127.0.0.1:5500/index.html)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A browser
+- A Github account
+### Setup
+
+Clone this repository to your desired folder:
+
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+
+
+### Install
+
+No need of installation.
+
+### Usage
+
+To run the project, execute the following command:
+
+Click the html file.
+
+### Run tests
+
+To run tests, run the following command:
+
+You can use a linter of your choice that is suitable for HTML and CSS.
+
+### Deployment
+
+You can deploy this project using:
+
+No need for deployments.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+👤 **Author**
+
+- GitHub: [@baezluise](https://github.com/baezluise)
+- Twitter: [@twitterhandle](https://twitter.com/baezluise1)
+- LinkedIn: [Luis Enrique Báez Méndez](https://www.linkedin.com/in/luis-enrique-b%C3%A1ez-m%C3%A9ndez-641917159/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+No futures features are planned for this project.
```

Comment:
- [ OPTIONAL] Please make sure to always add one or two future features instead of leaving them blank :wink: